libs/src/services/creatorNode/index.js

```diff
@@ -752,6 +779,77 @@ class CreatorNode {
     }
   }
 
+  async _uploadResumableTrackFile (file, onProgress) {
+    await this.ensureConnected()
+    const { headers } = this.createFormDataAndUploadHeaders(file)
+    const url = this.creatorNodeEndpoint + '/track_content_upload'
+
+    try {
+      return await this.startResumableUpload(file, url, headers, onProgress)
+    } catch (e) {
+      this._handleErrorHelper(e, url, headers['X-Request-ID'])
+    }
+  }
+
+  // https://github.com/tus/tus-js-client/blob/master/docs/api.md
+  async startResumableUpload (file, url, headers, onProgress) {
+    const randomFileName = uuid()
+    return new Promise((resolve, reject) => {
+      const upload = new tus.Upload(file, {
+        // Endpoint is the upload creation URL from your tus server
+        endpoint: url,
+        // Retry delays will enable tus-js-client to automatically retry on errors
+        retryDelays: [0, 3000, 5000, 10000, 20000],
```

Comment:
nice! did we come up with this or is this some standard they have? maybe link if it is

Reply:
this was provided in the example code they provided! the API reference is linked in the comment above this method 
```
// https://github.com/tus/tus-js-client/blob/master/docs/api.md
```